### PR TITLE
Adds UI for tickets/order route

### LINKED ISSUE #113: Add orders to tickets route (UI only)
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

Add Orders as sub route of tickets.

screenshot from 2017-06-02 13-29-13

- Located at `tickets/orders`

Please have a look at #97, before starting the work on the issue.

### PR DESCRIPTION
**Pull request by CosmicCoder96**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Adds the `tickets/order` route UI

#### Changes proposed in this pull request:

image

Fixes #113

### CODE REVIEW

File: app/templates/events/view/tickets/orders.hbs [hunk 1]
```diff
@@ -0,0 +1,142 @@
+<div class="ui stackable grid">
+  <div class="row">
+    <div class="eight wide column">
+      <h2 class="ui header">
+        {{t 'View Orders'}}
+      </h2>
+    </div>
+    <div class="eight wide {{unless device.isMobile 'right aligned'}} column">
+      {{#ui-dropdown class="selection"}}
```
Comment (geekyd) [inline app/templates/events/view/tickets/orders.hbs]: Please use single quotes inside helpers, follow https://github.com/fossasia/open-event-frontend/blob/development/docs/style-guide.md

File: app/templates/events/view/tickets/orders.hbs [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+            <i class="file pdf outline icon"></i>
+            {{t 'PDF'}}
+          </div>
+          <div class="item" data-value="CSV">
+            <i class="file excel outline icon"></i>
+            {{t 'CSV'}}
+          </div>
+        </div>
+      {{/ui-dropdown}}
+      {{#ui-dropdown class="selection"}}
```
Comment (geekyd) [inline app/templates/events/view/tickets/orders.hbs]: Please use single quotes inside helpers, follow https://github.com/fossasia/open-event-frontend/blob/development/docs/style-guide.md

**Comment by CosmicCoder96** [comment]
@niranjan94 @geekyd please review

**Comment by CosmicCoder96** [comment]
Deployment link : https://pr-123-opev-frontend.herokuapp.com/
@geekyd Please review.

**Comment by CosmicCoder96** [comment]
Heroku Deployment Link : 
https://pr-123-opev-frontend.herokuapp.com/events/963d3ba5/tickets/orders
@geekyd Please review.

**Comment by CosmicCoder96** [comment]
@geekyd Done 👍 Have made the dropdowns compact.

**Pull request merged**